packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -131,6 +131,8 @@ export interface VideoGalleryStrings {
   /* @conditional-compile-remove(spotlight) */
   /** Menu text shown in Video Tile contextual menu to stop spotlight on local user's video tile */
   stopSpotlightOnSelfVideoTileMenuLabel: string;
+  /** String for the attendee role */
+  attendeeRole: string;
```

Comment:
Should this be optional? Or behind a beta flag?

Reply:
Yes, CC flag was missed!